symbiflow-toolchain-xray/condarc

```diff
@@ -0,0 +1,5 @@
+channels:
+  - m-labs
```

Comment:
Why is `m-labs` needed here?

Reply:
It is not. It is a leftover. I removed it.